install/install.md

```diff
@@ -47,16 +47,42 @@ Use the selector tool below to select your preferred method, packages, and envir
+```
+conda create: error: argument --solver: invalid choice: 'libmamba' (choose from 'classic')
+```
 
-<br/>
+To resolve this error please follow one of these steps:
+- If the Conda installation is version `22.11` or newer, run: `conda install -n base conda-libmamba-solver`
+- If the Conda installation is older than `22.11`, please update your [Conda or Miniconda to the latest version](https://conda.io/projects/conda/en/stable/user-guide/install/index.html){: target="_blank"}
+- Use [Mamba directly](https://mamba.readthedocs.io/en/latest/installation.html){: target="_blank"}
+- Use the classic solver by removing `--solver=libmamba` from the `conda create` command provided by the selector
+
+<i class="fas fa-info-circle"></i> CUDA 12.0 ARM packages are not yet available:<br/>
+Conda-forge does not yet support the minimum required glibc (2.32) for CUDA 12 on ARM. For ARM support, please use CUDA 11.
+
+<i class="fas fa-info-circle"></i> At the time of writing, there is no stable CUDA 12 release of PyTorch: <br/>
+PyTorch currently only has nightly builds for CUDA 12.1, stable builds are limited to CUDA 11. <br/>
+The installation method below may allow RAPIDS CUDA 12.0 packages to coexist with PyTorch CUDA 12.1 nightly packages if there is a hard-requirement of CUDA 12 but it is currently unsupported:
+```
+mamba create -n rapids-pytorch-cu12 -c rapidsai -c pytorch-nightly -c conda-forge -c nvidia rapids={{ site.data.releases.stable.version }} cuda-version=12.0 pytorch pytorch-cuda=12.1
+```
 
 ### **Docker Issues**
-<i class="fas fa-info-circle"></i> Jupyter Lab is not accessible:<br/>
-If the server has not started or needs to be restarted / stop, use the included [start/stop script](#docker-startstop). Note this may change in the near future releases.
+<i class="fas fa-exclamation-triangle"></i> RAPIDS `23.08` brings significant Docker changes. <br/>
+To learn more about these changes, please see the [RAPIDS Container README](https://hub.docker.com/r/rapidsai/rapidsai){: target="_blank"}. Some key notes below:
+- `Development` images are no longer being published, in the coming releases RAPIDS will roll out [Dev Containers](https://code.visualstudio.com/docs/devcontainers/containers){: target="_blank"} for development
+   - See cuSpatial for an example and information on [RAPIDS' usage of Dev Containers](https://github.com/rapidsai/cuspatial/tree/main/.devcontainer){: target="_blank"}
+- All images are Ubuntu-based
+   - CUDA 11.2 images are Ubuntu `20.04`
+   - All other images are Ubuntu `22.04`
+- All images are multiarch (x86_64 and ARM)
+   - CUDA 12 is not yet supported when using Docker images on ARM architecture
+- The `Base` image starts in an ipython shell
+   - To run bash commands inside the ipython shell prefix the command with `!`
+   - To run the image without the ipython shell add `/bin/bash` to the end of the `docker run` command
```

Comment:
Should we link the RSN (https://docs.rapids.ai/notices/rsn0030/) or issue (https://github.com/rapidsai/docker/issues/539) that outlines all of the changes?

Reply:
Yeah I'll link that issue, it's very thorough and easy to read (ty!)